sql/moz-fx-data-shared-prod/monitoring_derived/duplicate_rate_v1/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+---
+friendly_name: Duplicate Rate
```

Comment:
Renaming to something like "Live Duplicate Rate" might maybe make it clearer that this data is only for live tables?

Reply:
the results table name should be like `live_duplicate_rate_v1` too?